### DISCUSSION TITLE
Can I Retrieve m and u Estimates from the Saved Model?

### DISCUSSION THREAD

**Discussion by meharc** [moj-analytical-services/splink#2088] [Q&A] [answered]
I trained a Splink model on Spark, but I can't find any API methods to extract m and u estimates, except for getting charts.

**Comment by RobinL** [comment, marked as answer]
You can retrieve the settings as a dict here, which contains the m and I values;
https://github.com/moj-analytical-services/splink/blob/bfaf8d142f7aad03f281b6324dd530e202eaec9a/splink/linker.py#L3693
